### PR TITLE
[textanalytics] v3.1 preview.5 generation

### PR DESCRIPTION
**Pull request by kristapratico**

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py [hunk 1]
```diff
@@ -305,6 +305,8 @@ def lro_get_next_page(lro_status_callback, first_page, continuation_token, show_
     parsed_url = urlparse(continuation_token)
     job_id = parsed_url.path.split("/")[-1]
     query_params = dict(parse_qsl(parsed_url.query.replace("$", "")))
+    if "showStats" in query_params:
+        query_params.pop("showStats")
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py:309]: preview.4 didn't seem to send "showStats" in the nextLink. It does get sent in preview.5, so we need to pop the camel cased version or else it gets sent to requests and we get an "unexpected keyword argument 'showStats'" TypeError.

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py [hunk 2]
```diff
@@ -656,9 +656,6 @@ def analyze_sentiment(  # type: ignore
             :class:`~azure.ai.textanalytics.SentenceSentiment` objects
             will have property `mined_opinions` containing the result of this analysis. Only available for
             API version v3.1-preview and up.
-        :keyword str string_index_type: Specifies the method used to interpret string offsets.  Possible values are
-            'UnicodeCodePoint', 'TextElements_v8', or 'Utf16CodeUnit'.  The default value is 'UnicodeCodePoint'.
-            Only available for API version v3.1-preview and up.
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py:661]: we have `string_index_type` duplicated in the docstring so I deleted one

File: sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py [hunk 3]
```diff
@@ -26,7 +26,7 @@
 # pre-apply the client_cls positional argument so it needn't be explicitly passed below
 TextAnalyticsClientPreparer = functools.partial(_TextAnalyticsClientPreparer, TextAnalyticsClient)
 
-
+@pytest.mark.skip("404 Not Found")
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py:29]: healthcare endpoint doesn't seem deployed yet?

File: sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze.py [hunk 4]
```diff
@@ -185,6 +185,9 @@ def test_bad_credentials(self, client):
                 ],
                 polling_interval=self._interval(),
             )
+
+    @pytest.mark.skip("Throws 400 on POST: (InvalidRequest) Job task parameter value bad is not supported for model-version "
+                 "parameter for job task type NamedEntityRecognition. Supported values latest,2019-10-01,2020-07-01")
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze.py:190]: need to follow up on this behavior change

File: sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py (hunk 3 above)
Comment (iscai-msft) [reply sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py:29]: yeah that seems to be the case, ok with skipping for now

**Review by iscai-msft — APPROVED** [review]
wow thank you so much krista you're a queen

**Pull request merged**